lib/models/device.coffee

```diff
@@ -51,6 +51,10 @@ getDeviceModel = (deps, opts) ->
 
 	exports = {}
 
+	# Infer dashboardUrl from apiUrl if former is undefined
+	if not dashboardUrl?
+		dashboardUrl = apiUrl.replace(/api/, 'dashboard')
```

Comment:
The apiUrl is required param, we should throw (if we don't) as early as possible - in the very beginning of the SDK factory.